addons/l10n_id_efaktur_coretax/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_uom_code_user,access_uom_code_user,model_l10n_id_efaktur_coretax_uom_code,,1,0,0,0
+access_product_code_user,access_product_code_user,model_l10n_id_efaktur_coretax_product_code,,1,0,0,0
```

Comment:
Don't define access rights without any groups :scream: 
Use base.group_user if you want to give access to all users.